internal/api/backresthandler.go

```diff
@@ -909,6 +911,21 @@ func (s *BackrestHandler) GetSummaryDashboard(ctx context.Context, req *connect.
 	return connect.NewResponse(response), nil
 }
 
+func humanTime(d time.Duration) string {
```

Comment:
super nit: let's return the duration as a uptimeMillis (int64 milliseconds is the common time resolution used in most places in backrest), this also makes it more useful for scripting.

We can reuse formatting logic already implemented in js in the webui, there's a utility for this is available in https://github.com/garethgeorge/backrest/blob/main/webui/src/lib/formatting.ts#L91-L118